src/elm/Page/Device.elm

```diff
@@ -544,7 +555,7 @@ update session msg model =
                     "Warning"
 
                 body =
-                    "This will remove the current device credential secret from Astarte, forcing the device to register again and store its new credentials secret. Continue?"
+                    Html.text "This will remove the current device credential secret from Astarte, forcing the device to register again and store its new credentials secret. Continue?"
```

Comment:
why converting the body type from string to html?

Reply:
Because the ConfirmModal now accepts html content instead of a simple string, in order to a more flexible modal body.